cli/types/cypress.d.ts

```diff
@@ -1826,11 +1826,6 @@ declare namespace Cypress {
      */
     prevUntil<E extends Node = HTMLElement>(element: E | JQuery<E>, filter?: string, options?: Partial<Loggable & Timeoutable>): Chainable<JQuery<E>>
 
-    /**
```

Comment:
I also took this opportunity to remove these types. We won't officially add them until we are ready to release the experiment